mint/PlotlyGraph.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+    <plotly :data="dataSeries" :layout="layout" :display-mode-bar="false"></plotly>
+</template>
+<script lang="ts">
+    import Vue from "vue"
+    import {Plotly} from "vue-plotly"
+
+    export default Vue.extend<any, any, any, any>({
+        props: ["settings", "columns", "dataSet", "data", "layout"],
+        components: {
+            Plotly
+        },
+        computed: {
+            dataSeries() {
```

Comment:
Sorry, I don't quite understand what's going on here. This is making the distinction between barchart type data and line graph type data based on its shape? What's the difference between data and dataset? Maybe a couple of brief comments would be useful.

Reply:
Agree, these should be renamed

Reply:
(data and dataSet are actually metadata and data, respectively, nothing to do with chart type)